src/adv.py

```diff
@@ -66,34 +74,95 @@
 #
 # If the user enters "q", quit the game.
 
-valid_cmds = ['s', 'n', 'e', 'w', 'q']
-
 game = True
 
 while game:
+    print(f"\n=================================================================================")
+    print("\n")
     # print the current room name and description
     print(
         f"You are currently in the {player.current_room.name}. {player.current_room.description}.")
+    print("\n")
     # print the items in a room
     if len(player.current_room.items) > 0:
         print("This room has the following items:")
-        for item in player.current_room.items:
-            print(f"{item.name}: {item.description}")
+        for i, item in enumerate(player.current_room.items, start=1):
+            print(
+                f"{i}) {item.name}: {item.description}")
+
     else:
-        print("This room has no items.")
+        print("There are no items here.")
+
+    print("\n")
 
     # prompt player for direction
-    cmd = input("Enter a direction (n, s, w, e) to move, or 'q' to quit: ")
+    cmd = input(
+        "What would you like to do? Select 'c' for a list of commands: ").lower()
 
-    if cmd in valid_cmds:
+    valid_cmds = ['s', 'n', 'e', 'w', 'q', 'c',
+                  'get', 'take', 'drop', 'i', 'inventory']
+    action = cmd.split()[0]
+
+    # if the first word entered is a valid command
+    if action in valid_cmds:
         direction = f"{cmd}_to"
         # grabs current_room of player, looks for direction attribute, defaults to None if invalid.
         new_room = getattr(player.current_room, direction, None)
```

Comment:
Good use of `getattr()`. You may also use `hasattr()` which returns a boolean of if an object has the attribute.